test/cql-pytest/test_tablets.py

```diff
@@ -100,6 +100,20 @@ def test_alter_changes_initial_tablets(cql, skip_without_tablets):
         assert res.initial_tablets == 2
 
 
+def test_alter_changes_initial_tablets_short(cql, skip_without_tablets):
```

Comment:
As usual, I want to complain about the missing comment that says what this test is about, and that it reproduces a specific issue.
Yes, a year from now someone can spend an hour in "git blame" archeology to figure this out. But a comment is so much better.